### DISCUSSION TITLE
Failed to load static props (client and server side errors)

### DISCUSSION THREAD

**Discussion by hamzalatif-e2x** [vercel/next.js#37503] [Help]
Hi,

I get an error when i try to navigate to a page (/home) in my nextjs app. It gives the following error on the client side in the browser:

<img width="1660" alt="Screenshot 2022-06-07 at 11 39 16" src="https://user-images.githubusercontent.com/61588905/172360928-597a910e-9bd5-4c81-8bb6-74d9aeaa01c1.png">

On the server side I get the following errors after receiving the request:

`TypeError: Cannot read properties of null (reading 'useContext')
    at Object.exports.useContext (/opt/node_modules/react/cjs/react.production.min.js:24:118)
    at Head (/opt/node_modules/next/dist/shared/lib/head.js:176:34)
`

`
TypeError: Cannot read properties of null (reading 'useContext')
    at Object.exports.useContext (/opt/node_modules/react/cjs/react.production.min.js:24:118)
    at Html (/opt/packages/app/.next/server/pages/_document.js:214:73)
`

And this results in a 500 Internal Server Error response.

Any ideas what is missing from my config?

I am using the `outputStandalone` config option with a custom server definition.

If I use the generated server.js file in the `.next/standalone` directory then everything works fine.

**Comment by AkramUlAdib** [comment]
did you got any solution?

**Comment by MatteoGauthier** [comment]
Long-standing Next.js brainfuck, in fact that's not really a Next.js issue, because you must wrap your React app with a ErrorBoundary, I think this should be added by default by Next IMO
